models/cont_delay_connection.h

```diff
@@ -33,7 +33,7 @@
 namespace nest
 {
 
-/* BeginUserDocs: synapse, continuous delay
+/* BeginUserDocs: synapse, connection with delay, continuous
```

Comment:
Please don't change tags in this PR. This is for another time.
```suggestion
/* BeginUserDocs: synapse, continuous delay
```